modules/test/playwright/tests/object-web/main/objectFields.spec.ts

```diff
@@ -932,6 +932,28 @@ test.describe('Manage objectFields through Objects Admin UI', () => {
 		).toBeDisabled();
 	});
 
+	test('can create custom object field in a system object definition', async ({
+		modelBuilderDiagramPage,
+		objectFieldsPage,
+		page,
+	}) => {
+		await objectFieldsPage.goto('Account');
+
+		const objectFieldLabel = `formula${getRandomInt()}`;
+
+		await objectFieldsPage.addObjectField({
+			formulaFieldOutput: 'Integer',
+			objectDefinitionNodes:
+				modelBuilderDiagramPage.objectDefinitionNodes,
+			objectFieldBusinessType: 'Formula',
+			objectFieldLabel,
+		});
+
+		await expect(page.getByText(objectFieldLabel)).toBeVisible();
+
+		await objectFieldsPage.deleteObjectField(true, -1);
+	});
+
```

Comment:
can we sort this tests alphabetically, please? yours should be the first one in this case. thank you!!